python/graphstorm/inference/np_infer.py

```diff
@@ -122,8 +125,6 @@ def infer(self, loader, save_embed_path, save_prediction_path=None,
                 ntype_emb = embs[ntype]
             embeddings = {ntype: ntype_emb}
 
-            device = th.device(f"cuda:{self.dev_id}") \
-                    if self.dev_id >= 0 else th.device("cpu")
             save_gsgnn_embeddings(save_embed_path,
                 embeddings, self.rank, th.distributed.get_world_size(),
                 device=device,
```

Comment:
Can you check if lp_infer.py and ep_infer.py have the same problem?

Reply:
ep_infer.py has the same issue. Fixed.

lp_infer.py does not have the issue.